runtime/kusama/src/governance/old.rs

```diff
@@ -116,8 +116,8 @@ parameter_types! {
 	pub const DesiredMembers: u32 = 19;
 	pub const DesiredRunnersUp: u32 = 19;
 	pub const MaxVotesPerVoter: u32 = 16;
-	pub const MaxVoters: u32 = 10 * 1000;
-	pub const MaxCandidates: u32 = 1000;
+	pub const MaxVoters: u32 = 64;
```

Comment:
This is not good -- why is it changing?

Reply:
In fact, instead, you should put a warning here telling parachains they should put small values here, and revet Polkadot to using the larger values.

Reply:
Reverted to the previous values.